src/python/gudhi/sklearn/cubical_persistence.py

```diff
@@ -98,7 +92,7 @@ def __transform(self, cells):
     def transform(self, X, Y=None):
         """Compute all the cubical complexes and their associated persistence diagrams.
 
-        :param X: List of cells filtration values (`numpy.reshape(X, newshape, order='C'` if `newshape` is set with a tuple of ints).
+        :param X: List of cells filtration values.
         :type X: list of list of float OR list of numpy.ndarray
```

Comment:
"list of list of float" can also be list of list of list of ... of float. I think list of array-like would work, if you don't want to limit to list of ndarray.

Reply:
Ok for list of array-like on [c0caed4](https://github.com/GUDHI/gudhi-devel/pull/856/commits/c0caed4de511df2695079eaf93b28bed860d59b7). I don't know if we should precise float ?